File: aula03/hello.f90
! Este programa deve ler um valor numérico (decimal, real) da entrada padrão
! e imprimir na saída padrão o seno deste valor

PROGRAM hello

	IMPLICIT NONE
	REAL(KIND=8) :: r, s
	WRITE(*,*) "Olá mundo!"

    ! Pedir para um número ser digitado
	WRITE(*,*) "Digite um número real: "
	READ(*,*) r
	s = sin(r);
	WRITE(*,*) "O seno de ", r , " é ", s

END PROGRAM hello

File: aula03/hello.py
import math

def hello(name):
    print("Olá, ", name)

name = input("Digite seu nome: ")

hello(name)

r = float(input("\nDigite um númetro real: "))

s = math.sin(r)
c = math.cos(r)
print("O seno de ", r , " é ", s)
print("O cosseno de ", r , " é ", c)

